README.md

```diff
@@ -1,2 +1,77 @@
 # kube-sqs-autoscaler
-Kubernetes pod autoscaler based on queue size in AWS SQS
+Kubernetes pod autoscaler based on queue size in AWS SQS. It periodically retrieves the number of messages in your queue and scales pods accordingly.
+
+## Setting up
+Setting up kube-sqs-autoscaler requires two steps:
+1) Deploying it as an incluster service in your cluster
+2) Adding AWS permissions so it can read the number of messages in your queues.
+
+### Deploying kube-sqs-autoscaler
+Deployin kube-sqs-autoscaler should be as simple as applying this deployment:
+```
+---
```

Comment:
If you follow the triple-backticks with the keyword `yaml`, the yaml will look nicer on Github.